### PR TITLE
Connect to API

### PR DESCRIPTION
**Pull request by tonnymuchui**
- Connected an existing API via Redux and used the data to display in a UI. :heavy_check_mark:

### CODE REVIEW

File: src/redux/books/books.js [hunk 1]
```diff
@@ -1,28 +1,68 @@
...
+    });
+    builder.addCase(addBook.fulfilled, (state, action) => {
+      state.entities.push({
+        id: action.payload.item_id,
+        title: action.payload.title,
+        author: action.payload.author,
+      });
+    });
+    builder.addCase(removeBook.fulfilled, (state, action) => {
+      // eslint-disable-next-line no-param-reassign
```
Comment (zieeco) [inline src/redux/books/books.js]: - Please, let's kindly remove the eslint disable and refactor the code to conform to the linter as the helps in writing clean and maintainable and this is not ideal to disable the linter in microverse. Please, let's improve on this.

File: src/redux/books/books.js [hunk 2]
```diff
@@ -1,28 +1,68 @@
...
+    builder.addCase(fetchBooks.fulfilled, (state, action) => {
+      const newBookList = [];
+      Object.entries(action.payload).forEach((item) => {
+        newBookList.push({
+          id: item[0],
+          title: item[1][0].title,
+          author: item[1][0].author,
+        });
+      });
+      // eslint-disable-next-line no-param-reassign
```
Comment (zieeco) [inline src/redux/books/books.js]: - Please, let's kindly remove the eslint disable and refactor the code to conform to the linter as the helps in writing clean and maintainable and this is not ideal to disable the linter in microverse. Please, let's improve on this.

**Review by zieeco — CHANGES_REQUESTED** [review]
Hi @tonnymuchui ,


Great job so far! 👏 
You've done well, however, there are some issues that you still need to work on to go to the next project, but you are almost there!


## To highlight 🎯

- No linter errors.✔️
- Nice design. ✔️
- Descriptive PR. ✔️


## Required Changes ♻️
_Check the comments under the review._





## Optional suggestions
- N/A


**_Every comment with the **[OPTIONAL]** prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._**


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by emmiiorji — APPROVED** [review]
Hi @tonnymuchui, 

# Status: Approved ✅

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

congratulations

## Good Highlights✔️

- Requested changes have been implemented

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**